### ISSUE TITLE
Multimethod object conversions throwing TypeError

### ISSUE THREAD

**Issue by matthewturk** [mandiant/Ghidrathon#82]
Hello!  I'm working to port over a script to use Ghidrathon.  The original script I had included a variant of this:

```python
regs = [program.getLanguage().getRegister(argument["reg"].upper())]
```

and then used this to create an instance of `ghidra.program.model.listing.VariableStorage` with:

```python
vs = VariableStorage(program, [regs])
```

What I'm running into is two-fold.  The first is that as-is, this doesn't catch the multimethod quite right, and it throws an error that it expects a list of `Varnode` objects.  But, by changing the invocation to:

```python
vs = VariableStorage(program, *regs)
```

I believe it correctly selects the appropriate constructor.  However, the error it brings back is odd to me:

```
Traceback (most recent call last):
  File "/home/mturk/.ghidra/.ghidra_10.3.1_PUBLIC/Extensions/Ghidrathon/data/python/jepeval.py", line 74, in jepeval
    more_input_needed = _jepeval(line)
                        ^^^^^^^^^^^^^^
  File "/home/mturk/.ghidra/.ghidra_10.3.1_PUBLIC/Extensions/Ghidrathon/data/python/jepeval.py", line 49, in _jepeval
    exec(compile(line, "<string>", "single"), globals(), globals())
  File "<string>", line 1, in <module>
TypeError: Error converting parameter 2: Expected [Lghidra.program.model.lang.Register; but received a ghidra.program.model.lang.Register.
```

A few things strike me as odd about this -- `isinstance(regs[0], ghidra.program.model.lang.Register)` returns `True`, and additionally, I'm not sure where the stray `[L` is coming from in the string.  I was hoping someone might have some insight into where the type checks are going awry, and how I can either fix it locally or if it's possible to work on and submit a patch to do so.  Thanks!

**Comment by mike-hunhoff** [comment]
Thank you for reporting @matthewturk ! This was actually an issue with Jep that was fixed in the [recent Jep 4.2 release](https://github.com/ninia/jep/releases/tag/v4.2.0). Here is an example of your use case using the [latest Ghidrathon 3.0.2 release](https://github.com/mandiant/Ghidrathon/releases/tag/v3.0.2) (released earlier today):
```

   _____ _     _     _           _   _                 
  / ____| |   (_)   | |         | | | |                
 | |  __| |__  _  __| |_ __ __ _| |_| |__   ___  _ __  
 | | |_ | '_ \| |/ _` | '__/ _` | __| '_ \ / _ \| '_ \ 
 | |__| | | | | | (_| | | | (_| | |_| | | | (_) | | | |
  \_____|_| |_|_|\__,_|_|  \__,_|\__|_| |_|\___/|_| |_|
                                                       
Python 3.12.1 Interpreter for Ghidra 10.4. Developed by FLARE.

>>> from ghidra.program.model.listing import VariableStorage
>>> regs = [currentProgram().getLanguage().getRegister("EAX")]
>>> regs
[<ghidra.program.model.lang.Register object at 0x000002DE13E8BDC0>]
>>> vs = VariableStorage(currentProgram(), *regs)
>>> vs
<ghidra.program.model.listing.VariableStorage object at 0x000002DE08B36B90>
>>> vs.isRegisterStorage()
True
>>> vnodes = [vs.getFirstVarnode()]
>>> vnodes
[<ghidra.program.model.pcode.Varnode object at 0x000002DE13CD2C50>]
>>> vs = VariableStorage(currentProgram(), *vnodes)
>>> vs
<ghidra.program.model.listing.VariableStorage object at 0x000002DE13E8A9B0>
>>> vs.getRegister()
<ghidra.program.model.lang.Register object at 0x000002DE08B36B90>
>>> vs.getRegister().toString()
'EAX'
```
Please reopen this issue if you continue to experience this issue with [Ghidrathon 3.0.2](https://github.com/mandiant/Ghidrathon/releases/tag/v3.0.2).

The `[Lghidra.program.model.lang.Register;` notation is specific to the Java Virtual Machine where `[L` describes an array of `ghidra.program.model.lang.Register` instances (more generically, `[L<ClassName>;` where `<ClassName>`, e.g.  `ghidra.program.model.lang.Register`, describes the type of the instances that are stored in the array).

**Comment by matthewturk** [comment]
It works!  Thank you so much.
